python/ray/ray_constants.py

```diff
@@ -48,6 +48,8 @@ def env_bool(key, default):
 # we attempt to start the service running at this port.
 DEFAULT_PORT = 6379
 
+RAY_ADDRESS_ENVIRONMENT_VARIABLE = "RAY_ADDRESS"
+
```

Comment:
Why do we need this? Can't we already use RAY_ADDRESS?

Reply:
I can do that in a follow up PR, there are only 8 occurrences of it (including one place where it is already defined as a constant.